### PR TITLE
tmuxinator 0.15.0 (new formula)

### PR DESCRIPTION
**Pull request by simobasso**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
ref: https://github.com/tmuxinator/tmuxinator
ref: https://github.com/Homebrew/homebrew-core/pull/34043
ref: https://github.com/tmuxinator/tmuxinator/issues/563

### CODE REVIEW

**Review by javian — CHANGES_REQUESTED** [review]

File: Formula/tmuxinator.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  test do
+    version_output = shell_output("#{bin}/tmuxinator version")
+    assert_match "tmuxinator #{version}", version_output
+
+    list_output = shell_output("#{bin}/tmuxinator list")
+    assert_match "tmuxinator projects:", list_output
+
+    system bin/"tmuxinator", "new", "test"
+    list_output = shell_output("#{bin}/tmuxinator list")
+    assert_match "tmuxinator projects:", list_output
```
Comment (igas) [inline Formula/tmuxinator.rb]: after you created `test` project you are not checking it here.

**Comment by simobasso** [comment]
Hi @javian and @igas, thanks for the reviews. I have just updated the pr with your suggestions. 🎉

File: Formula/tmuxinator.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+class Tmuxinator < Formula
+  desc "Manage complex tmux sessions easily"
+  homepage "https://github.com/tmuxinator/tmuxinator"
+  url "https://github.com/tmuxinator/tmuxinator/archive/v0.15.0.tar.gz"
+  sha256 "e5c121126aebe3afc758c0561b8ef05508712a799d3821453063b87445806ed4"
+  depends_on "tmux"
```
Comment (javian) [inline Formula/tmuxinator.rb]: There should be an empty line here.

File: Formula/tmuxinator.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "tmuxinator.gemspec"
```
Comment (javian) [inline Formula/tmuxinator.rb]: This creates an issue that the dependencies are not versioned. If you look at the `travis.rb` formula you can see that the gems are listed as resources which the gem is then build from and something similar would be needed here to make reproducible builds. Can you arrange that ?

**Review by javian — CHANGES_REQUESTED** [review]

**Pull request closed without merging**